lecturer/junit_intro/src/test/java/by/jrr/start/ApplicationTest.java

```diff
@@ -10,10 +10,13 @@
     @Test
     public void main() {
 
-        int expected = 1;
+        int expected = 2;
 
         int actual = Integer.valueOf("2");
 
         Assert.assertEquals("Failed", expected, actual);
```

Comment:
Assert.assertEquals("Failed");
  Use meaningful messages.

Reply:
I fixed it in my new pull request https://github.com/javaGuruBY/rpaS3/pull/195/files.